rpc/shared/shared/src/main/scala/io/udash/rpc/AutoUdashRPCFramework.scala

```diff
@@ -0,0 +1,16 @@
+package io.udash.rpc
+
+import com.avsystem.commons.serialization._
+
+import scala.language.postfixOps
+
+trait AutoUdashRPCFramework extends UdashRPCFramework {
+  type RawValue = String
+
+  implicit val RawValueCodec: GenCodec.Auto[RawValue] =
```

Comment:
Why is this needed? `GenCodec.Auto` always tries to wrap already existing `GenCodec`.